src/components/Hero/Hero.mdx

```diff
@@ -484,6 +485,47 @@ should not be used at the same time.
   language="jsx"
 />
 
+## Custom Image Component
+
+Instead of passing a path for the `imageProps.src` prop, you can pass a custom image component in using the `imageProps.component` prop.
+Note that this only impacts the "campaign" variant, and the custom component will only replace the foreground image, since the background image is set using `background-image`.
+
+<Source
+  code={`
+import Image from "next/image";
```

Comment:
To hammer the point of using a custom image, what do you think about updating the import statement to something like this?
```
import Image as NextImage from "next/image";
```
And then, of course, updating the reference in the code below.